PHPGraphQLArticle.md

```diff
@@ -212,55 +202,49 @@ query UserPosts{
+The query is a lot easier to understand than those cumbersome REST calls above. We can look at it and see that we want the posts from user 45, and the last 5 comments of each post with the id and name of the user of each comment. Simple.
 
-In the server, it is also very nice to implement. Indeed, the resolver there does not need to know that we will want that nesting. I mean, it does not need to know we want the post WITH the comments. Because the comments have their own resolver and the GraphQL layer will call it on demand. 
+In the server, it is also very easy to implement. The resolver there does not need to know we want the post WITH the comments. This is because the comments have their own resolver and the GraphQL layer will call it on demand. 
 
-So, in practice, we don't need to do anything different to return the post than what we need to do to return it nested with it's comments. 
+In short, we don't need to do anything different to return the post than when we return it nested with it's comments. 
 
 *So, server code gets cleaner and well organized.* 
 
-These are only a small subset of the improvements I can see on GraphQL over REST. I hope they are enought to encourage your reading of this article. In the following sections, looking at a real world example, you will sure be able to understand a lot more about GraphQL and it's benefits. 
+In the following sections, where we look at a real world example, you will start understanding the deeper intricacies of GraphQL.
 
 ### PHP
 
-I usually use PHP and Symfony on the backend, so that was my natural choice to the server. I went in a quest to see if I could find good libs in PHP to help on the job. And, lucky me, thanks fot these both projects ([1](https://github.com/webonyx/graphql-php),[2](https://github.com/Youshido/GraphQL)) I found mature libs that do their job in a excelent manner. 
+I usually use PHP and Symfony on the backend, so that was my natural choice for this server. I went hunting to see if I could find any good libs in PHP. Thanks to these projects ([1](https://github.com/webonyx/graphql-php),[2](https://github.com/Youshido/GraphQL)) I have access to some mature libs. 
 
-I started using Youshido's lib and later migrated to Webonyx. To integrate with symfony, I used the [OverblogGraphQLBundle](https://github.com/overblog/GraphQLBundle). Having that GraphQL server layer in place, meanwhile I was evolving with the app, I tried a lot of layer compositions and code organizations to find a balanced and clean one. 
+I started using Youshido's lib but later migrated to Webonyx. To integrate with Symfony, I used the [OverblogGraphQLBundle](https://github.com/overblog/GraphQLBundle). Having that GraphQL server layer in place, meanwhile I was evolving with the app, I tried a lot of layer compositions and code organizations to find a balanced and clean one. ISSUES WITH THIS PARA
```

Comment:
What issues? Let me try to improve it...
"
I started using Youshido's lib but later migrated to Webonyx. To integrate it with Symfony, I used the [OverblogGraphQLBundle](https://github.com/overblog/GraphQLBundle) to make the server available. Having that GraphQL server layer in place, meanwhile I was evolving with the app, I tried a lot of layer compositions and code organizations to finally find a balanced and clean one. 
"
If it's worse than before, just throw it away. :-)